config/defaults.mk

```diff
@@ -291,6 +292,10 @@ ifeq ($(MFEM_USE_OCCA),YES)
   OCCA_LIB := -Wl,-rpath,$(OCCA_DIR)/lib -L$(OCCA_DIR)/lib -locca
 endif
 
+UMPIRE_DIR = @MFEM_DIR@/../umpire
+UMPIRE_OPT = -I$(UMPIRE_DIR)/include
+UMPIRE_LIB = -L$(UMPIRE_DIR)/lib -lumpire -lumpire_op -lumpire_resource -lumpire_strategy -lumpire_tpl_judy -lumpire_util
+
```

Comment:
Are all of this libraries really necessary... we can't just link with `-lumpire`?

Reply:
Thanks for taking a look! Currently these are all necessary. That's slated to change very soon in Umpire. After that you're correct, all we'll need is `-lumpire`. Stay tuned!